docs/getting-started/peoplefinder/create-policy.mdx

```diff
@@ -18,27 +18,27 @@ To get started, we'll create a policy repository and a single policy instance th
 
 After creating your organization, navigate to the Images tab.
 
-<img src="/images-tab.png" width="200px" />
+<img src="/images-tab.png" />
```

Comment:
don't you want width="500px" or something like this here? otherwise it'll be huge?